src/world/area_kkj/kkj_19/AF7950.c

```diff
@@ -1,11 +1,20 @@
 #include "kkj_19.h"
 
-static char* N(exit_str_0) = "kkj_10";
-static char* N(exit_str_1) = "kkj_00";
+typedef struct {
+    /* 0x00 */ char unk_00[4];
```

Comment:
fields are itemID, flagIndex, overrideDescMsg

Reply:
That's very precise! How could you tell? Did you check the unmatched functions in the file to see what the struct is used for?

Reply:
At first, I was seeing if this was effect data, so I checked the calling script in the Star Rod dump for this map. Seeing 80240E10 is an array, I read through the asm in the dump to figure out what the fields were for and then ran the map in an emulator with a debugger to confirm.

Reply:
Thanks for the explanation. I haven't used Star Rod before but I may take a look later. I'll probably learn a couple of things.